app/jobs/appointment_notification/worker.rb

```diff
@@ -41,22 +41,25 @@ def send_message(notification, communication_type)
       NotificationService.new
     end
 
-    if communication_type == "missed_visit_whatsapp_reminder"
+    case communication_type
+    when "whatsapp"
       notification_service.send_whatsapp(
         notification.patient.latest_mobile_number,
         notification.localized_message,
         callback_url
       ).tap do |response|
         metrics.increment("sent.whatsapp")
       end
-    else
+    when "sms"
       notification_service.send_sms(
         notification.patient.latest_mobile_number,
         notification.localized_message,
         callback_url
       ).tap do |response|
         metrics.increment("sent.sms")
       end
+    else
+      raise StandardError, "#{self.class.name} is not configured to handle communication type #{communication_type}"
```

Comment:
This looks good, but can you create a custom Exception class to raise?  Maybe `UnknownCommunicationType`.

Reply:
Will do.